src/lib/local-storage-utils.js

```diff
@@ -0,0 +1,53 @@
+/* @flow */
+import constants from './constants';
+import generate from './generate-id';
+
+const { storage, sevenDays } = constants;
+
+/* Returns an existing cartId or null */
+export const getCartId = () => {
```

Comment:
this function is exported for the sole reason for testing? Its not used anywhere, and I can't think of a use case other than being a helper function in here. What do you think about not exporting it altogether (and then remove the test for `getCartId`

Reply:
This is used by both 'resetExpiredCartId' and 'getOrCreate'. It's exported for the sake of testing.